### ISSUE TITLE
missing logs for background services

### ISSUE THREAD

**Issue by james-lawrence** [flutter/flutter-intellij#2996]
## Steps to Reproduce

logs of background services are entirely missing from android studio. with no way of displaying them.

## Version info

/shared/james-lawrence/development/flutter/bin/flutter doctor --verbose
[✓] Flutter (Channel beta, v1.0.0, on Linux, locale en_US.UTF-8)
    • Flutter version 1.0.0 at /shared/james-lawrence/development/flutter
    • Framework revision 5391447fae (4 weeks ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /shared/james-lawrence/Android/Sdk
    • Android NDK at /shared/james-lawrence/Android/Sdk/ndk-bundle
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /shared/james-lawrence/development/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Android Studio (version 3.2)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 31.3.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] Android Studio (version 3.2.)
    • Android Studio at /shared/james-lawrence/development/android-studio
    • Flutter plugin version 31.3.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] Connected device (2 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 9 (API 28) (emulator)
    • Android SDK built for x86 • emulator-5556 • android-x86 • Android 9 (API 28) (emulator)

• No issues found!
Process finished with exit code 0

**Comment by pq** [comment]
Are these background services implementing in Flutter plugins?  It would be great if we could reproduce the issue on our end to see if this is more of a framework or IDE issue.  Thanks in advance for any more context!

**Comment by james-lawrence** [comment]
nope, its just a GRPC service that I then call using dart's grpc library. small update: if flutter settings are changed to verbose logging the logs appear for the service. but they should appear by default imo.
